### ISSUE TITLE
Fertilizer Attachment

### ISSUE THREAD

**Issue by ApatheticSchizoid** [StrangeLoopGames/EcoSuggestions#1552]
It'd be nice to have a tractor attachment that fertilizers your fields while getting as close to 100% nutrients as possible. It'd make large scale farms so much easier.

**Comment by Sybbyl0127** [comment]
I agree with this on so many levels, I have massive farms on my current server and the one thing I have the most trouble with is keeping them all up on nutrients, especially since I have a double tiered big bluestem farm for fiber->mortar supply that drops nutrients at twice the speed.

Keeping them all managed is a massive chore because it takes *so* long to go from one plot to the next applying fertilizers
